test/testData.ts

```diff
@@ -402,16 +321,20 @@ export const mixedResult = {
     {
       Id: '07Mxx00000ErgiHUAR',
       QueueItemId: '709xx000001IlUMQA0',
+      // @ts-ignore
```

Comment:
Let's use `@ts-expect-error` for these so that we remember to remove them once these Types are updated to be `string | null`.